index.html

```diff
@@ -1,54 +1,269 @@
+            </div>
+          </section>
+
+        </div>
+
+        <div class="card">
+
+          <img class="cardimg" src="./images/Snapshoot Portfolio.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Tonic</p>
+            <div class="canopy">
+              <p>CANOPY</p>
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads;
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See Proyect</button></a>
+            </div>
+          </section>
+
+        </div>
+
+        <div class="card">
+
+          <img class="cardimg" src="./images/profesional.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Tonic</p>
+            <div class="canopy">
+              <p>CANOPY</p>
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads;
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See Proyect</button></a>
+            </div>
+          </section>
+
+        </div>
+      </section>
+
+      <section class="aboutmyself">
+        <h2>About Myself</h2>
+
+        <article class="offer">
+          <p>
+            Hello I’m a software developer! I can
+            help you build a product , feature or
+            website. Look through some of my
+            work and experience! If you like what
+            you see and have a project you need
+            coded, don’t hesitate to contact me.
+          </p>
+        </article>
+        <section class="contact">
+          <p>LET’S CONNECT</p>
+        </section>
+
+        <section class="socialmedia-botton">
+          <ul>
+            <li><img src="./images/git.png" alt=""></li>
+            <li><img src="./images/I.png" alt="">
+              <img src="./images/N.png" alt="">
+            <li><img src="./images/hand.png" alt=""></li>
+            <li class="twitter"><img src="./images/twitter.png" alt=""></li>
+            </li>
+            <li><img src="./images/m.png" alt=""></li>
+          </ul>
+        </section>
+
+        <div class="get">
+          <a href="#"><button>Get my resume</button></a>
+        </div>
+
+
+        <section class="lenguage">
+
```

Comment:
- [ ] As suggested by the previous reviewer, kindly fix that typo to `language`, as it is grammatically correct, and it will help people better understand your code. 👍